src/Argon/Argon.php

```diff
@@ -55,7 +55,7 @@ public static function parseWithCurrentLocale(
      */
     public static function createFromFormatWithCurrentLocale(
         string $format,
-        string $time = null,
+        string|null $time = null,
```

Comment:
I prefer this format - the only time I would use `|null` is at the end of a union type (ie. `string|int|null`).
```suggestion
        ?string $time = null,
```